lib/Runtime/PlatformAgnostic/RuntimePlatformAgnosticPch.h

```diff
@@ -44,6 +44,7 @@ class Throw
 public:
     static bool ReportAssert(const char* fileName, unsigned int lineNumber, const char* error, const char* message);
     static void LogAssert();
+    static void __declspec(noreturn) FatalInternalError();
```

Comment:
??

Reply:
I believe this is required to use AssertOrFailFast from within PlatformAgnostic